File: Java/Java source-code/min.java


public class min {
    public static double min_m(Double[] elements) {
        int size = elements.length;
        double min = elements[size - 1];
        for (int i = size - 1; --i >= 0; ) {
            if (elements[i] < min) {
                min = elements[i];
            }
        }
        return min;
    }
}

File: Methods/Methods-Python/min.py
def min(data):
    size = len(data)

    if(size == 0):
        raise NameError('IligalArgumentException')

    elements = data.copy()
    minimoun = elements[size - 1]

    for i in range(size - 1, -1, -1):
        if elements[i] < min:
            min = elements[i]

    return min